src/OpenRiaServices.Tools/Test/ServerClassLib/ServerClassLib.csproj

```diff
@@ -1,18 +1,22 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
-    <TargetFramework>net472</TargetFramework>
+    <TargetFrameworks>net472;net6.0</TargetFrameworks>
     <!-- We compile with Release settings to test PDB reader,
          though we pretend it is the Debug build just to simplify
          a debug build of the solution
     -->
     <DefineConstants>TRACE</DefineConstants>
-    <DebugType>portable</DebugType>
+    <DebugType>pdbonly</DebugType>
     <Optimize>true</Optimize>
   </PropertyGroup>
-  <ItemGroup>
+  <!--<ItemGroup>
```

Comment:
Remove?

Reply:
Look into the new PackageReferences, are they really needed ?